databases/week2/Homework/Homework_tables.sql

```diff
@@ -0,0 +1,152 @@
+USE Clinic_scheduling;
+
+CREATE TABLE
+  Clinics (
+    clinic_id INT NOT NULL AUTO_INCREMENT,
+    Clinic_name VARCHAR NOT NULL,
+    Clinic_address VARCHAR NOT NULL,
+    PRIMARY KEY (clinic_id)
+  );
+
+-- a USER can either have work schedule for certain date or a regular working schedule for certain weekdays
+-- hence different kind of time-slots
+CREATE TABLE
+  Working_hours (
+    date DATE,
+    weekday CHAR(3),
+    begins DATETIME NOT NULL,
+    ends DATETIME NOT NULL,
+    slot_id INT NOT NULL AUTO_INCREMENT,
+    PRIMARY KEY (slot_id)
+  );
+
+CREATE TABLE
+  Users (
+    user_id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR NOT NULL,
+    sex VARCHAR,
+    address VARCHAR,
+    phone INT NOT NULL,
+    email VARCHAR NOT NULL,
+    notes VARCHAR,
+    PRIMARY KEY (user_id)
+  );
+
+CREATE TABLE
+  Medical_forms (
+    Form_id INT NOT NULL AUTO_INCREMENT,
+    Form_name VARCHAR NOT NULL,
+    Form_link VARCHAR NOT NULL,
+    Form_description VARCHAR NOT NULL,
+    PRIMARY KEY (Form_id)
+  );
+
+-- each doctor and several other user roles have specific schedules
+CREATE TABLE
+  Doctors_have_working_hours (
```

Comment:
it is better to use the same naming across the database, i.e. you don't have a table with 'Doctors' but 'Users'.
Also, it is a practice not to use verbs in the table names but to describe the table relations. In this case, the table name should be 'user_working_hours' or just 'schedules'

Reply:
Really great suggestion - it's much easier to read and use.